src/Microsoft.Framework.PackageManager/Restore/RestoreCommand.cs

```diff
@@ -136,6 +145,136 @@ public RestoreCommand(IApplicationEnvironment env)
             }
         }
 
+        private async Task<bool> RestoreForInstall(string packagesDirectory)
```

Comment:
Why was this broken out into RestoreForInstall? Wasn't it part of a single RestoreForProject earlier?